src/plugins/paginate/paginate-en.hbs

```diff
@@ -0,0 +1,839 @@
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Accusantium quisquam ea fugiat optio!</h3>
+				<p>Repudiandae quisquam animi veritatis. Sit odio minus doloribus suscipit similique fugit voluptatem dicta sint. Alias!</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Voluptas fuga officiis fugiat. Totam.</h3>
+				<p>Fugiat neque ipsa placeat harum eaque veniam. Molestiae esse dolore temporibus cum illum mollitia aliquid?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Accusantium nobis laudantium tempora repudiandae?</h3>
+				<p>Blanditiis, obcaecati quas sed consequuntur deserunt culpa odit, sint tempore soluta temporibus expedita veniam! Impedit?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Debitis sint vel voluptas illo.</h3>
+				<p>Autem nam rerum omnis praesentium dicta, maxime maiores rem, perferendis, possimus qui vero ad sapiente.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Quam dolores modi mollitia nesciunt.</h3>
+				<p>Suscipit rerum sunt, vel quam labore doloribus illo adipisci nam ea similique impedit maiores id.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Possimus dolore nulla velit doloribus!</h3>
+				<p>Officiis pariatur nobis sequi quod culpa enim natus est praesentium, dolor sint voluptas, dignissimos eum.</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Assumenda, praesentium id. Alias, sint.</h3>
+				<p>Mollitia tempora quae optio dicta obcaecati. Consectetur nisi iste sunt, eos perspiciatis maxime at dignissimos?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Earum officia saepe voluptatibus dolor!</h3>
+				<p>Eum non natus ullam, adipisci expedita, amet iusto quam, officiis ut error necessitatibus harum voluptatem?</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Impedit natus aperiam nemo adipisci?</h3>
+				<p>Illum dolore ipsam corporis atque quisquam cupiditate quod aliquid quaerat, saepe architecto aspernatur ratione autem!</p>
+			</div>
+		</article>
+		<article class="col-sm-6 col-md-4 col-lg-3">
+			<div class="well">
+				<h3 class="mrgn-tp-0">Assumenda dignissimos beatae doloremque dolores.</h3>
+				<p>Aliquid temporibus sunt quis illum, voluptatem atque. Blanditiis voluptatum iure cupiditate recusandae hic sint aperiam.</p>
+			</div>
+		</article>
+	</div>
+</section>
+
+<section id="filteredEventsList">
```

Comment:
In this example the pagination are outside the scope of the section element. Would be preferable to add the pagination plugin via an inner div inside the section?